script/lib/create-debian-package.js

```diff
@@ -200,9 +200,32 @@ module.exports = function(packagedAppPath) {
   );
 
   console.log(`Generating .deb file from ${debianPackageDirPath}`);
-  spawnSync('fakeroot', ['dpkg-deb', '-b', debianPackageDirPath], {
-    stdio: 'inherit'
-  });
+
+  // don't compress by default to speed up build
+  let compressionLevel = 0;
+  let compressionType = 'none';
+  if (process.env.IS_RELEASE_BRANCH || process.env.IS_SIGNED_ZIP_BRANCH) {
+    compressionLevel = 6;
+    compressionType = 'xz';
+  }
+  // use sudo if available to speed up build
+  let sudoCommand = 'fakeroot';
+  if (process.env.CI || (process.getuid && process.getuid() === 0)) {
```

Comment:
> `process.getuid() === 0`

This seems to check if the build script is running as root. I don't think folks should run the build script as root/with `sudo`... So this condition won't be hit by most users who are being responsible with security on their systems.

Perhaps don't check for that?

Reply:
To replace `fakeroot` with `sudo` which is way faster.